foodsaving/pickups/tests/test_models.py

```diff
@@ -0,0 +1,112 @@
+from dateutil import rrule
+from dateutil.relativedelta import relativedelta
+from django.db import DataError
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.test import TestCase
+from django.utils import timezone
+from datetime import datetime
+
+from foodsaving.pickups.factories import PickupDateFactory
+from foodsaving.pickups.models import PickupDateSeries, PickupDate, Feedback
+from foodsaving.users.factories import UserFactory
+from foodsaving.stores.factories import StoreFactory
+
+
+class TestFeedbackModel(TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Ah, you introduced the old style again that Nick removed with #441 - could you fix this? See #425 for more details.

Reply:
Sorry, this was the result of git's auto-merge, that's why I missed it... I randomly checked some of the auto-merged files but unfortunately not this one.

It's fixed now.